admin/actions/class-webchangedetector-settings-action-handler.php

```
@@ -54,6 +59,244 @@ public function handle_save_group_settings( $data ) {
+		$group_ids     = $all_groups[ $group_key ];
+
+		if ( empty( $group_ids ) ) {
+			return array(
+				'success' => false,
+				'message' => __( 'No registered groups found.', 'webchangedetector' ),
+			);
+		}
+
+		// First, save to the current site's group normally (this handles auto-update website settings too).
+		if ( $is_monitoring ) {
+			$single_result = $this->handle_monitoring_settings( $data );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$single_result'. (undefined)

(UnusedLocalVariable)